src/index.js

```diff
@@ -1,37 +1,60 @@
 import { Service } from 'feathers-memory';
-import debounce from 'lodash/debounce';
 
 class LocalStorage extends Service {
   constructor(options = {}) {
     super(options);
     this.storageKey = options.name || 'feathers';
-    this.storage = options.storage || window.LocalStorage;
-    this.store = JSON.parse(this.storage.getItem(this.storageKey) || '{}');
-    this.write = debounce(() => 
-      this.storage.setItem(this.storageKey, JSON.stringify(this.store)),
-      options.throttle || 100
-    );
+    this.storage = options.storage || window.localStorage;
+    this.throttle = options.throttle || 200;
+    this.store = null;
+  }
+  
+  ready() {
+    if(!this.store) {
+      return Promise.resolve(this.storage.getItem(this.storageKey))
+        .then(str =>JSON.parse(str || '{}'))
```

Comment:
I think we might have to have a try catch here in case someone pre-loads their localstorage data with invalid JSON.